src/main/java/com/mazawrath/beanbot/commands/admin/AdminPostChangeLogCommand.java

```diff
@@ -30,23 +34,23 @@ public void onCommand(String[] args, DiscordApi api, ServerTextChannel serverTex
             serverTextChannel.sendMessage(getRecentChangeLog());
             serverTextChannel2.sendMessage("Changelog sent to " + serverTextChannel.getName() + ".");
         });
+
+        Sentry.clearContext();
     }
 
     private String getRecentChangeLog() {
```

Comment:
See if you can use something like this in the future
https://developer.github.com/v3/repos/releases/#get-the-latest-release